### PR TITLE
update Gateway Discovery tutorial with client verification

### PR DESCRIPTION
**Pull request by czeslavo**
### Description

 We should recommend using client verification when GD deployment is used. Issue: https://github.com/Kong/kubernetes-ingress-controller/issues/3833

### Testing instructions

Netlify link: https://deploy-preview-5415--kongdocs.netlify.app/

### Checklist 

- [x] Review label added 
- [x] PR pointed to correct branch (`main` for immediate publishing, or a release branch: e.g. `release/gateway-3.2`, `release/deck-1.17`)

### CODE REVIEW

File: app/_src/kubernetes-ingress-controller/guides/using-gateway-discovery.md [hunk 1]
```diff
@@ -32,70 +32,106 @@ This guide will be also using [`stern`][stern-gh] for easy pod logs querying.
...
+  --set admin.enabled=true \
+  --set admin.type=ClusterIP \
+  --set admin.clusterIP=None \
+  --set admin.tls.client.secretName=admin-api-ca-cert \
+  --set replicaCount=2
+```
+
+which will deploy {{site.base_gateway}} with 2 replicas, without {{site.kic_product_name}}.
+
+Once installed, set an environment variable, $PROXY_IP with the External IP address of
```
Comment (pmalek) [inline app/_src/kubernetes-ingress-controller/guides/using-gateway-discovery.md:112]:
```suggestion
Once installed, set an environment variable, `$PROXY_IP` with the External IP address of
```

**Review by pmalek — APPROVED** [review]
👍

File: app/_src/kubernetes-ingress-controller/guides/using-gateway-discovery.md [hunk 2]
```diff
@@ -32,70 +32,106 @@ This guide will be also using [`stern`][stern-gh] for easy pod logs querying.
...
+> and `ingressController.adminApi.tls.client.secretName` should be used 
+> for that purpose in the {{site.kic_product_name}} release, and 
+> `admin.tls.client.secretName` or `admin.tls.client.caBundle` in the 
+> {{site.base_gateway}} release.
+> For more information, see the [helm chart's readme][kong-chart].
+
 Now we're ready to deploy controller's helm release:
 
-```
+```terminal
```
Comment (Guaris) [inline app/_src/kubernetes-ingress-controller/guides/using-gateway-discovery.md:70]:
```suggestion
```sh
```

File: app/_src/kubernetes-ingress-controller/guides/using-gateway-discovery.md [hunk 3]
```diff
@@ -32,70 +32,106 @@ This guide will be also using [`stern`][stern-gh] for easy pod logs querying.
...
   --set deployment.kong.enabled=false \
   --set proxy.nameOverride=${GATEWAY_RELEASE_NAME}-kong-proxy \
   --set replicaCount=2
 ```
 
-At this point you should be able to see both deployments ready:
+At this point you should be able to see a {{site.kic_product_name}} deployment with 
+2 replicas, waiting for {{site.base_gateway}} to be deployed:
 
+```terminal
```
Comment (Guaris) [inline app/_src/kubernetes-ingress-controller/guides/using-gateway-discovery.md:85]:
```suggestion
```sh
```

File: app/_src/kubernetes-ingress-controller/guides/using-gateway-discovery.md [hunk 4]
```diff
@@ -32,70 +32,106 @@ This guide will be also using [`stern`][stern-gh] for easy pod logs querying.
...
+$ kubectl get deployment -n kong
+NAME              READY   UP-TO-DATE   AVAILABLE   AGE
+controller-kong   0/2     2            2           1m
 ```
+
+### {{site.base_gateway}}
+
+As we enabled TLS client verification for the Admin API service, we need to provide a CA
```
Comment (Guaris) [inline app/_src/kubernetes-ingress-controller/guides/using-gateway-discovery.md:93]:
```suggestion
After enabling TLS client verification for the Admin API service, you need to provide a CA
```

File: app/_src/kubernetes-ingress-controller/guides/using-gateway-discovery.md [hunk 5]
```diff
@@ -32,70 +32,106 @@ This guide will be also using [`stern`][stern-gh] for easy pod logs querying.
...
+certificate to the {{site.base_gateway}} deployment. We can do that by setting
```
Comment (Guaris) [inline app/_src/kubernetes-ingress-controller/guides/using-gateway-discovery.md:94]:
```suggestion
certificate to the {{site.base_gateway}} deployment. You can do that by setting
```

File: app/_src/kubernetes-ingress-controller/guides/using-gateway-discovery.md [hunk 6]
```diff
@@ -32,70 +32,106 @@ This guide will be also using [`stern`][stern-gh] for easy pod logs querying.
...
+`admin.tls.client.secretName` to the CA certificate Secret's name, which we set in the
```
Comment (Guaris) [inline app/_src/kubernetes-ingress-controller/guides/using-gateway-discovery.md:95]:
```suggestion
`admin.tls.client.secretName` to the CA certificate Secret's name, which was set in the
```

File: app/_src/kubernetes-ingress-controller/guides/using-gateway-discovery.md [hunk 7]
```diff
@@ -32,70 +32,106 @@ This guide will be also using [`stern`][stern-gh] for easy pod logs querying.
...
+{{site.kic_product_name}} deployment.
+
+Now we can deploy {{site.base_gateway}}. In order to do that we can run:
```
Comment (Guaris) [inline app/_src/kubernetes-ingress-controller/guides/using-gateway-discovery.md:98]:
```suggestion
Now you can deploy {{site.base_gateway}}. In order to do that we can run:
```

File: app/_src/kubernetes-ingress-controller/guides/using-gateway-discovery.md [hunk 8]
```diff
@@ -32,70 +32,106 @@ This guide will be also using [`stern`][stern-gh] for easy pod logs querying.
...
+  --set admin.tls.client.secretName=admin-api-ca-cert \
+  --set replicaCount=2
+```
+
+which will deploy {{site.base_gateway}} with 2 replicas, without {{site.kic_product_name}}.
+
+Once installed, set an environment variable, $PROXY_IP with the External IP address of
+the `gateway-kong-proxy` service in `kong` namespace:
+
+```terminal
```
Comment (Guaris) [inline app/_src/kubernetes-ingress-controller/guides/using-gateway-discovery.md:115]:
```suggestion
```sh
```

File: app/_src/kubernetes-ingress-controller/guides/using-gateway-discovery.md [hunk 9]
```diff
@@ -32,70 +32,106 @@ This guide will be also using [`stern`][stern-gh] for easy pod logs querying.
...
+$ export PROXY_IP=$(kubectl get -o jsonpath="{.status.loadBalancer.ingress[0].ip}" service -n kong gateway-kong-proxy)
+```
+
+At this point you should be able to see both deployments ready:
+
+```terminal
```
Comment (Guaris) [inline app/_src/kubernetes-ingress-controller/guides/using-gateway-discovery.md:121]:
```suggestion
```sh
```

File: app/_src/kubernetes-ingress-controller/guides/using-gateway-discovery.md [hunk 10]
```diff
@@ -108,14 +144,14 @@ Additional replicas, will:
...
-```
+```terminal
 $ kubectl scale deployment -n kong gateway-kong --replicas 4
 $ kubectl scale deployment -n kong controller-kong --replicas 3
 ```
 
 At this point we should see 4 instances of gateway and 3 instances of controller:
 
-```
+```terminal
```
Comment (Guaris) [inline app/_src/kubernetes-ingress-controller/guides/using-gateway-discovery.md:154]:
```suggestion
```sh
```

File: app/_src/kubernetes-ingress-controller/guides/using-gateway-discovery.md [hunk 11]
```diff
@@ -203,7 +239,7 @@ spec:
 With those manifests applied we can observe controller logs for entries that
 indicate successful configuration of all the discovered Gateways:
 
-```
+```terminal
```
Comment (Guaris) [inline app/_src/kubernetes-ingress-controller/guides/using-gateway-discovery.md:242]:
```suggestion
```sh
```

File: app/_src/kubernetes-ingress-controller/guides/using-gateway-discovery.md [hunk 12]
```diff
@@ -214,7 +250,7 @@ controller-kong-545d798874-q6h7m ingress-controller time="2023-03-02T15:11:42Z"
 
 At this point we should be able to access the `/echo` endpoint from our `htptbin` service:
 
-```
+```terminal
```
Comment (Guaris) [inline app/_src/kubernetes-ingress-controller/guides/using-gateway-discovery.md:253]:
```suggestion
```sh
```

**Review by Guaris — APPROVED** [review]
Some small grammar and formatting corrections.

**Pull request merged**